### ISSUE TITLE
Typo in haddock documentation of ToSeriesData class

### ISSUE THREAD

**Issue by msakai** [maoe/influxdb-haskell#27]
`toSeriesColumn` is used as a method name, but it should be `toSeriesColumns` to match actual implementation.

```
diff --git a/src/Database/InfluxDB/Encode.hs b/src/Database/InfluxDB/Encode.hs
index d812268..6b28c90 100644
--- a/src/Database/InfluxDB/Encode.hs
+++ b/src/Database/InfluxDB/Encode.hs
@@ -28,7 +28,7 @@ class ToSeries a where
 -- > data EventType = Login | Logout
 -- >
 -- > instance ToSeriesData Event where
--- >   toSeriesColumn _ = V.fromList ["user", "type"]
+-- >   toSeriesColumns _ = V.fromList ["user", "type"]
 -- >   toSeriesPoints (Event user ty) = V.fromList [toValue user, toValue ty]
 -- >
 -- > instance ToValue EventType
```

**Comment by maoe** [comment]
Thanks for reporting! Would you like to send a pull request?

**Comment by msakai** [comment]
Okay, I made a pull request.

**Comment by maoe** [comment]
Fixed in #28. I'll make a release later.
